client-js/app/scripts/app.ts

```diff
@@ -23,10 +25,11 @@ window.addEventListener('WebComponentsReady', function() {
     page(e.detail.path);
   });
 
-  var socket = new registry.JsonRpcSocket('ws://localhost:9000/api/socket2');
+  var socket = new clientRpc.JsonRpcSocket('ws://localhost:9000/api/socket2');
+  socket.prefix = "org.labrad."
   var reg = new registry.RegistryServiceJsonRpc(socket);
+  var echo = new echoServer.EchoServiceJsonRpc(socket);
```

Comment:
I think I misunderstood what you mean by prefix. I thought you meant ```org.labrad``` as prefix, but you mean, ```org.labrad.registry``` etc? If we were sending a request to an address other than ```org.labrad``` I would say open a new socket, but if you mean the whole things then I agree.